src/olympia/addons/serializers.py

```diff
@@ -421,7 +421,9 @@ def fake_object(self, data):
                     display_username=persona_data['author'],
                     header=persona_data['header'],
                     footer=persona_data['footer'],
-                    persona_id=1 if persona_data['is_new'] else None,
+                    # "New" Persona do not have a persona_id, it's a relic from
+                    # old ones.
+                    persona_id=0 if persona_data['is_new'] else 42,
```

Comment:
and `1` is too boring? 😃 
Ideally all those image files should be [converted](#2695) so we don't have to have an `is_new` property at all.